doc/sphinx/user_manual/tools/experimental/complps2pbes.rst

```diff
@@ -0,0 +1,8 @@
+.. index:: complps2pbes
+
+.. _tool-complps2pbes:
+
+complps2pbes
+============
+
+.. include:: man/complps2pbes.txt
```

Comment:
As I explained in a comment on the main thread, this is actually generated automatically by the custom extension. It just does it in the source now, which is tracked by Git. I felt that this is an okay way to do it, but I'm open to changing it.

Reply:
I see now. As also noted in another comment, I think it's nicest to leave the source tree untouched when building documentation.

Reply:
The only way I see to do that is to copy the entire source tree, make modifications to some of the files (man pages) and finally run Sphinx on that copy. That seems overkill, but it is definitely possible. Would that be preferable?

Reply:
Maybe a good intermediate solution is to only copy the rst files related to tool files to the build directory?

Reply:
Copying only part of the files is not workable, because Sphinx scans an entire directory (and only one) for source files. I have fixed this by copying the entire source tree (see 65f3910356974449f363c4f3f063955a7829a6c4 and following commits).